Toric/ConvexAnalysis/Salient.lean

```diff
@@ -0,0 +1,24 @@
+/-
+Copyright (c) 2025 Aaron Liu. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Aaron Liu
+-/
+import Mathlib.Analysis.Convex.Cone.Pointed
+import Mathlib.Analysis.Convex.Exposed
+
+/-!
+# Salient Cones
+
+We prove some equivalent conditions for a cone to be salient.
+-/
+
+variable {E : Type*} [NormedAddCommGroup E] [TopologicalSpace E]
+  [InnerProductSpace ℝ E] (S : PointedCone ℝ E)
+
+theorem PointedCone.salient_tfae :
+  [
+    S.toConvexCone.Salient,
+    IsExposed ℝ S {(0 : E)},
```

Comment:
It goes the other way around (and I expect you don't need the type ascription, although I haven't tested)
```suggestion
    IsExposed ℝ {0} S,
```